packages/core/deploy/002_deploy_registry.js

```diff
@@ -1,30 +1,14 @@
-const { interfaceName } = require("@uma/common");
 const func = async function(hre) {
-  const { deployments, getNamedAccounts, web3 } = hre;
-  const { utf8ToHex, padRight } = web3.utils;
-  const { deploy, log, execute } = deployments;
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
 
   const { deployer } = await getNamedAccounts();
 
-  const deployResult = await deploy("Registry", {
+  await deploy("Registry", {
     from: deployer,
     args: [],
     log: true
   });
-
-  if (deployResult.newlyDeployed) {
```

Comment:
I removed all this Finder-pointing from the deployment scripts so that the deploy files wouldn't all be linked to the Finder. Practically, I was running each of these scripts individually and wanted to avoid having to deploy the Finder over and over again.